README.md

```diff
@@ -87,23 +95,12 @@ export REPLICATED_APP=...
 export REPLICATED_API_TOKEN=...
 ```
 
-`REPLICATED_APP` should be set to the app slug from the Settings page:
-
-<p align="center"><img src="./doc/REPLICATED_APP.png" width=600></img></p>
-
-Next, create a Service Account API token from the vendor portal under [Service Accounts](https://vendor.replicated.com/team/serviceaccounts):
-
-<p align="center"><img src="./doc/REPLICATED_API_TOKEN.png" width=600></img></p>
+You can find documentation on how to set environment variables [here](https://docs.replicated.com/reference/replicated-cli-installing#set-environment-variables)
 
 Ensure the token has the appropriate "Write" access in the selected [RBAC policy](https://vendor.replicated.com/team/policies) or you'll be unable create new releases. Once you have the values,
 set them in your environment.
 
-```
-export REPLICATED_APP=...
-export REPLICATED_API_TOKEN=...
-```
-
-You can ensure this is working with
+Once the environment variables are set, you can validate configuration by running the following command:
 
 ```
 replicated release ls
```

Comment:
This whole section about manifests and linting I'm not 100% sure is accurate for helm. This reads like it's talking about KOTS (with all the references to 'manifests' instead of talking about charts).

Have you run through this and is it for a native helm v2 install?